econplayground/api/urls.py

```diff
@@ -12,6 +13,8 @@
 router.register(r'graphs', GraphViewSet)
 router.register(r'submissions', SubmissionViewSet)
 router.register(r'topics', TopicViewSet)
+router.register(r'assignments', AssignmentViewSet)
+router.register(r'questions', QuestionViewSet)
```

Comment:
All this API view stuff can be removed, as the current plan is just to implement this in Django. I mean, we could use React, but that just adds unnecessary complexity, from my perspective.

Reply:
Already done when I removed the serializers

Reply:
Wait, do you mean the entire file or just the my additions?